sdks/python/setup.py

```diff
@@ -205,6 +205,34 @@ def generate_protos_first():
                        err.stderr)
 
 
+def generate_external_transform_wrappers():
+  try:
+    sdk_root = os.path.dirname(os.path.realpath(__file__))
+    if not os.path.exists(os.path.join(sdk_root, 'gen_xlang_wrappers.py')):
+      beam_root = os.path.join(sdk_root, 'apache_beam')
+      wrappers = list(find_by_ext(beam_root, '_et.py'))
+
+      if not wrappers:
+        warnings.warn(
+          'External transform wrappers have not been generated and the '
+          'generation script `gen_xlang_wrappers.py` cannot be found')
+      else:
+        warnings.warn('Skipping external transform wrapper generation as they '
+                      'are already generated.')
+      return
+    out = subprocess.run([
+      sys.executable,
+      os.path.join('gen_xlang_wrappers.py'),
+      '--cleanup',
+      '--input-expansion-services', 'standard_expansion_services.yaml',
+      '--output-transforms-config', 'standard_external_transforms.yaml'],
+      capture_output=True, check=False)
```

Comment:
```suggestion
      capture_output=True, check=True)
```

If False, it skips if there is an error in the subprocess.

Reply:
Thanks, set to True